src/conductor/checks.py

```diff
@@ -178,27 +178,34 @@ def exists(self):
         """
         item_id = None
         item_name = None
-        MetaResponse = namedtuple('MetaResponse', 'exists item_id item_name')
+        geometry_type = None
+        MetaResponse = namedtuple('MetaResponse', 'exists item_id item_name geometry_type')
 
         TableChecker.get_data(self, (self.original_table))
 
         if self.data is None:
-            self.data = MetaResponse(False, item_id, item_name)
+            self.data = MetaResponse(False, item_id, item_name, geometry_type)
 
-            return MetaResponse(False, item_id, item_name)
+            return MetaResponse(False, item_id, item_name, geometry_type)
 
-        item_id, item_name = self.data
+        item_id, item_name, geometry_type = self.data
         response = True
 
-        if not item_id and not item_name:
-            response = False
-        elif not item_id:
-            response = 'missing item id'
+        if not item_id and not item_name and not geometry_type:
+            return MetaResponse(False, item_id, item_name, geometry_type)
+
+        errors = []
+        if not item_id:
+            errors.append('missing item id')
         elif not item_name:
-            response = 'missing item name'
-        #: TODO: add geometry type
+            errors.append('missing item name')
+        elif not geometry_type:
+            errors.append('missing geometry type')
```

Comment:
This will only append one error (item id) even if all three are missing, no? Should the `elif`s be changed to `if`s?

Reply:
yes, yes they should.